subgraphs/uniswap-v2/src/common/constants.ts

```diff
@@ -0,0 +1,38 @@
+import { Address, BigDecimal, BigInt } from '@graphprotocol/graph-ts';
+
+export const PROTOCOL_ID = "1337"
+
+export const NETWORK_ETHEREUM = "ETHEREUM"
```

Comment:
You should update the named constants using the `namespaced` format here: https://github.com/messari/subgraphs/blob/40d5822bf6dbdbcae185a7ccdd77b526c9dc5cbd/subgraphs/yearn-v2/src/common/constants.ts#L7

Reply:
Fixed